README.md

```diff
@@ -29,12 +29,18 @@ This client assumes your app  has either `raven` or `sentry_sdk` installed
 [Sentry SDK](https://github.com/getsentry/sentry-python)
 
 
+## Upgrade to version 3.0.0 considerations
+
+The default ID field has been changed to `email_user_id`. Previously the `user_id` (guid) was the default field - see below for details on how to revert to `user_id` if needed.
+
+`MIGRATE_EMAIL_USER_ON_LOGIN` logic has been removed.
+
 ## Upgrade to version 1.0.0 considerations
 
 From version `1.0.0` the backend populates `User.USERNAME_FIELD` with the `user_id` rather than the `email`. This is
 to solve a bug affecting users with multiple email addresses.
 If `MIGRATE_EMAIL_USER_ON_LOGIN` is `True`, the authentication backend tries to migrate existing users.
-It is recommended to turn `MIGRATE_EMAIL_USER_ON_LOGIN` to `False` (defaults to `False`) if not needed or when all the users are migrated to avoid 
+It is recommended to turn `MIGRATE_EMAIL_USER_ON_LOGIN` to `False` (defaults to `False`) if not needed or when all the users are migrated to avoid
```

Comment:
I would remove the instructions relating to MIGRATE_EMAIL_USER_ON_LOGIN since you have removed the logic that relates to it. Perhaps put something in about how this setting can be removed from the project configuration.

Reply:
Yep this makes sense.  Maybe I could remove the whole "upgrade to 1.0.0 considerations" block?

Reply:
Sounds good to me.